src/api/app/decorators/statistics/maintenance_statistic_decorator.rb

```diff
@@ -0,0 +1,27 @@
+module Statistics
+  class MaintenanceStatisticDecorator < BaseDecorator
+    def to_hash_for_xml
```

Comment:
Why not using a `case` statement and only setting the differences?, instead of constructing the whole thing each time and needing to have a `default_hash` method

```ruby
def to_hash_for_xml
  result = { type: model.type, when: model.when }
  case model.type
  when :issue_created
    result.merge!({ name: model.name, tracker: model.tracker })
  when :review_accepted, :review_opened
    result.merge!({ who: model.who, id: model.id })
  end
  result
end
```